src/main/java/com/redhat/management/approval/EmailBody.java

```diff
@@ -48,10 +48,15 @@ public String getEmailTemplate() {
         values.put("contents", getRequestContentLines(requestContent));
 
         String webUrl = System.getenv("APPROVAL_WEB_URL");
-        String approveLink = webUrl + "/api/approval/v1.0/stageaction/" + approver.getRandomAccessKey();
+        // base path: /api/approval/v1.1
+        String basePath = System.getenv("APPROVAL_BASE_PATH");
+        // catalog base path: /ansible/catalog/approval
+        String catalogBasePath = System.getenv("CATALOG_BASE_PATH");
```

Comment:
`APPROVAL_UI_BASE_PATH`, or
`APPROVAL_UI_PATH` that includes `/requests/detail/`, or
make `APPROVAL_UI_PATH` to include a placeholder variable where you can replace with `getApprovalId()` instead of always appending to the end.

Reply:
I chose the 2nd way: `APPROVAL_UI_PATH` that includes `/requests/detail/`